### ISSUE TITLE
can't start detection with stream or movie

### ISSUE THREAD

**Issue by oob444** [cgtinker/BlendArMocap#71]
blender version is 3.0.1 
after installing everything I pres the start detection button and it throws an error into me:

Python: Traceback (most recent call last):
  File "C:\Users\дом\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\BlendArMocap-main\src\cgt_blender\interface\ui_operators.py", line 140, in modal
    running = self.detection_handler.detector.image_detection()
  File "C:\Users\дом\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\BlendArMocap-main\src\cgt_detection\detect_face.py", line 30, in image_detection
    with self.solution.FaceMesh(
  File "C:\Users\дом\Desktop\blender-3.0.1-windows-x64\3.0\python\lib\site-packages\mediapipe\python\solutions\face_mesh.py", line 94, in __init__
    super().__init__(
  File "C:\Users\дом\Desktop\blender-3.0.1-windows-x64\3.0\python\lib\site-packages\mediapipe\python\solution_base.py", line 253, in __init__
    validated_graph.initialize(
FileNotFoundError: The path does not exist.

location: <unknown location>:-1

pls help me

**Comment by cgtinker** [comment]
Seems like an error within mediapipe. Which version of Blendarmocap / operating system U are using?

**Comment by oob444** [comment]
BlendArMocap - release 152 (that's how the downloaded file named), and Windows 10

**Comment by cgtinker** [comment]
actually I've misread the log. it states that you started movie detection without selecting a movie file. can u send me the log when you start a stream? or, make sure to select a movie file when starting the movie detection

**Comment by oob444** [comment]
this error shows up when I start a stream: 

Python: Traceback (most recent call last):
  File "C:\Users\дом\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\BlendArMocap-main\src\cgt_blender\interface\ui_operators.py", line 140, in modal
    running = self.detection_handler.detector.image_detection()
  File "C:\Users\дом\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\BlendArMocap-main\src\cgt_detection\detect_pose.py", line 30, in image_detection
    with self.solution.Pose(
  File "C:\Users\дом\Desktop\blender-3.0.1-windows-x64\3.0\python\lib\site-packages\mediapipe\python\solutions\pose.py", line 146, in __init__
    super().__init__(
  File "C:\Users\дом\Desktop\blender-3.0.1-windows-x64\3.0\python\lib\site-packages\mediapipe\python\solution_base.py", line 253, in __init__
    validated_graph.initialize(
FileNotFoundError: The path does not exist.

location: <unknown location>:-1

And this error shows up when I start the movie detection:

Python: Traceback (most recent call last):
  File "C:\Users\дом\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\BlendArMocap-main\src\cgt_blender\interface\ui_operators.py", line 45, in execute
    selected_armature = user.selected_rig.name_full
AttributeError: 'NoneType' object has no attribute 'name_full'

location: <unknown location>:-1

But this error shows only after trying to transfer the animation from mp4

**Comment by cgtinker** [comment]
I think U tried to transfer while not having a generated rogify rig as target. You got to set the transfer target.

Besides, it seems like mediapipe has an issue in it self here, can U run as admin, uninstall the dependencies and reinstall? Then check if the error persists.

**Comment by oob444** [comment]
Nope. I did all this but the same error shows when I start a stream
And this error shows when I try to transfer animation from mp4 (now with selected armature)

Python: Traceback (most recent call last):
  File "C:\Users\дом\AppData\Roaming\Blender Foundation\Blender\3.0\scripts\addons\BlendArMocap-main\src\cgt_blender\interface\ui_operators.py", line 53, in execute
    col_mapping[col](armature, driver_objects)
KeyError: 'Collection'

location: <unknown location>:-1

**Comment by oob444** [comment]
Thank you for the help so far. How will I know that you've already done something to the stream? shall I simply check your ithub page sometimes or something like this?

**Comment by cgtinker** [comment]
You are welcome. 

I'll keep this issue open till its fixed. Seems to be a new issue with mediapipe and there aren't resources available yet.

**Comment by cgtinker** [comment]
the new version of mediapipe (0.9.1) has better packaging than (0.8.x), so this is resolved
